lib/hangman.rb

```diff
@@ -0,0 +1,54 @@
+class Word
```

Comment:
Generally the filename should be the same as the class name, ie. this should be `lib/word.rb`

Reply:
Thanks! File name and test file name now changed to match the class name